data-pipeline/src/health_metrics.rs

```diff
@@ -7,7 +7,7 @@ pub(crate) const STAT_SEND_TRACES: &str = "datadog.libdatadog.send.traces";
 pub(crate) const STAT_SEND_TRACES_ERRORS: &str = "datadog.libdatadog.send.traces.errors";
 pub(crate) const STAT_DESER_TRACES: &str = "datadog.libdatadog.deser_traces";
 pub(crate) const STAT_DESER_TRACES_ERRORS: &str = "datadog.libdatadog.deser_traces.errors";
-pub(crate) const STAT_SER_TRACES_ERRORS: &str = "datadog.libdatadog.ser_traces.errors";
+// pub(crate) const STAT_SER_TRACES_ERRORS: &str = "datadog.libdatadog.ser_traces.errors";
```

Comment:
Super minor nit, but I slightly prefer this. If you agree, you could do the same for the constants in `trace_exporter`.

```suggestion
#[warn(dead_code)]
pub(crate) const STAT_SER_TRACES_ERRORS: &str = "datadog.libdatadog.ser_traces.errors";
```